### ISSUE TITLE
Successfully published package is not available for download

### ISSUE THREAD

**Issue by jstritch** [atom/atom#25426]
### Prerequisites

* [X] Put an X between the brackets on this line if you have done all of the following:
    * Reproduced the problem in Safe Mode: <https://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode>
    * Followed all applicable steps in the debugging guide: <https://flight-manual.atom.io/hacking-atom/sections/debugging/>
    * Checked the FAQs on the message board for common solutions: <https://discuss.atom.io/c/faq>
    * Checked that your issue isn't already filed: <https://github.com/issues?utf8=✓&q=is%3Aissue+user%3Aatom>
    * Checked that there is not already an Atom package that provides the described functionality: <https://atom.io/packages>

### Description

I've attempted to publish an update to a package. Although the command line interface appears successful, the new version is not available for download.

I've made three attempts this morning. On the first, my token had expired and I needed to apm login with a refreshed token. The second and third attempts appeared fully successful from the command line. But when I view https://atom.io/packages the version from last month is shown. The package does not show as having an update available from within Atom Packages Updates.

I examined the Recently Updated section of https://atom.io/packages and checked the dates of several repositories.  The most recent I found was two days ago. No Atom package updates were published in the last two days?

My repository is https://github.com/jstritch/git-tabular-diff

### Steps to Reproduce

1. apm publish patch

**Expected behavior:**

1. A success indication from the command line.
1. The updated package is available for installation.

**Actual behavior:**

1. A success indication from the command line.
1. The previous version of the package remains available for installation.

**Reproduces how often:**

I updated the CHANGELOG.md and tried again with the same result.

### Versions

atom --version
Atom    : 1.60.0
Electron: 9.4.4
Chrome  : 83.0.4103.122
Node    : 12.14.1

apm --version
apm  2.6.2
npm  6.14.13
node 12.14.1 x64
atom 1.60.0
python
git 2.35.1

I am running: Kernel: 5.13.0-40-generic x86_64 bits: 64 Desktop: Cinnamon 5.2.7 Distro: Linux Mint 20.3 Una

**Comment by asiloisad** [comment]
I have the same problem, the published packages in last 4 days are not available in atom.io. My tokan has also expired, but all refresh attempts fail.

**Comment by ladar** [comment]
Is this your issue: #25417 and if so post there, and so we know it's still impacting you.

**Comment by jstritch** [comment]
No, I am seeing different symptoms - I do not get the 500 internal server error. I'm attaching some screen shots with additional explanation to help clarify what I'm observing.

On Saturday the Atom package page was displaying the previous version 1.1.1 after the publish.  I have checked occasionally since then and seen 1.1.4 (the correct version), the old version, and no version.  The displayed version changes. No version is the current situation, shown here:
![atom-package-page-displays-no-version](https://user-images.githubusercontent.com/66251356/165317806-94116822-8933-49d3-8e41-e12e564ddd6a.png)

Within atom itself, the previous version is displayed with no option to update, shown here: 
![atom-displays-previous-version](https://user-images.githubusercontent.com/66251356/165318293-1c130164-c4ad-444f-9443-572371566c0a.png)

The preferences update section also fails, as shown here:
![atom-displays-up-to-date-incorrectly](https://user-images.githubusercontent.com/66251356/165318519-dc63bba6-3ecb-410a-9fda-84dd8f785944.png)

GitHub actions was able to get the correct version several times earlier this morning, as shown here:
![github-actions-obtain-correct-version](https://user-images.githubusercontent.com/66251356/165318873-ed552112-887e-454c-8102-53ee4abb5f1c.png)

I uninstalled the package successsfully.  Attempting to reinstall resulted in the following: 
![atom-fails-to-install](https://user-images.githubusercontent.com/66251356/165322039-fda9af1f-9c8e-4e81-88d6-a774508d49db.png)

Please let me know if you have additional questions.

**Comment by jstritch** [comment]
After the last install failure screen shot above, installing from the command line installs the old version, 1.1.1.

**Comment by asiloisad** [comment]
It look like atom package system is broken from few days. There is no changes in "Recently Updated" and "Newest" sections at atom.io/packages. For example, the shell command `apm view image-paste` display version v0.3.3, while i have updated this pacakge to v0.3.7 without any error from `apm publish patch` (just check the github https://github.com/bacadra/atom-image-paste). The command `apm install bacadra/atom-image-paste` work correctly -- it install latest version v0.3.7.

**Comment by jstritch** [comment]
Problem remains. It may be affecting affecting many packages.

Possibly related to https://github.com/atom/atom/issues/25419.

**Comment by jstritch** [comment]
In the screen shot I just took, below, only two (2) packages show version information. Please explain.

![atom-packages](https://user-images.githubusercontent.com/66251356/165604299-80bac5af-d8c2-4ca1-847b-4c7a3e09462f.png)

**Comment by asiloisad** [comment]
are there any news in this topic? the atom.io/packages "Newest" and "Recently Updated" sections are still frozen and `apm publish` still doesn't work.

**Comment by asiloisad** [comment]
it look like the update system is still broken. the newest package in section has last update in 2019.

![image](https://user-images.githubusercontent.com/29706474/166078907-ddb616f3-b542-4139-b4b2-6638945ffdfd.png)

![image](https://user-images.githubusercontent.com/29706474/166078925-66d52dc9-09a1-4c89-a8f6-20256081617d.png)

**Comment by gossip1212** [comment]
This is an awesome motivating article.I am practically satisfied with your great work.You put truly extremely supportive data. Keep it up. Continue blogging. Hoping to perusing your next post   [https://talksgossip.com/](url)

**Comment by stephen730** [comment]
Just got the news they will sunset Atom. Terrible news, everybody. I just started using it and loved the application so far. It was something different to add to my aresenal of programs when choosing which one works best for each project. Sad to hear the language sunset being used. Hopefully this stuff goes away! Cheers!!

--[Stephen Littleton](https://StephenLittleton.com)

**Comment by stephen730** [comment]
[Here is a link to the official comment that I viewed.](https://github.blog/2022-06-08-sunsetting-atom/)
